### ISSUE TITLE
Keep @required annotations when generating route argument classes

### ISSUE THREAD

**Issue by kuhnroyal** [Milad-Akarie/auto_route_library#5]
My `@required` annotations from my page parameters get lost inside the generated arguments classes.

I think it would be great if they get generated as well, maybe even some automatically generated asserts for `@required` parameters.

**Comment by Milad-Akarie** [comment]
How did I miss the @required annotation!! Thanks for bringing that to my attention. I will make sure its included in the next build.

**Comment by Milad-Akarie** [comment]
done in version 0.2.0.

**Comment by ResoDev** [comment]
I'm using the latest version of the package on a clean project and this constructor
```dart
const ThirdPage({
  @required this.userName,
  @required this.points,
});
```

generated an Arguments class without @required annotations and the `hasInvalidArgs` method doesn't get `isRequired: true` passed into it.

```dart
//ThirdPage arguments holder class
class ThirdPageArguments {
  final String userName;
  final double points;
  ThirdPageArguments({this.userName, this.points});
}
```

**Comment by Milad-Akarie** [comment]
@ResoDev My bad! I used the wrong deprecated property replacement. 
I just uploaded the fix in version auto_route_generator 0.2.1+2.
Thanks for reporting.
